docs/src/global.css

```diff
@@ -35,6 +35,11 @@ html[theme="g90"] .code-override {
   overflow-x: auto;
 }
 
+/* Override to accommodate mandatory footer */
+.bx--header~.bx--side-nav {
+  height: calc(100% - 120px);
```

Comment:
The magic number may need some adjusting, as the side nav appears to be cut off.

<img width="994" alt="Screenshot 2024-07-17 at 8 45 21 AM" src="https://github.com/user-attachments/assets/03edc5e5-a2f2-496e-a017-780569638a34">

Reply:
<img width="1314" alt="Screenshot 2024-07-17 at 8 46 26 AM" src="https://github.com/user-attachments/assets/641999bd-4f65-421f-846e-965cdafca324">